circle.yml

```diff
@@ -1,7 +1,9 @@
 machine:
   php:
     # We support PHP 5.4 so choose the latest version supported by Circle.
-    version: 5.4.37
+    #version: 5.4.37
+    # BBS override
+    version:  5.6.17
```

Comment:
I would prefer just to make the change instead of having an outcommented configuration value.